src/include/distributed/version_compat.h

```diff
@@ -52,6 +53,7 @@
 #define PortalDefineQuerySelectCompat(a,b,c,e,f) PortalDefineQuery(a,b,c,"SELECT",e,f)
 #define getOwnedSequencesCompat(a,b) getOwnedSequences(a,b)
 #define ExplainOnePlanCompat(a,b,c,d,e,f,g,h) ExplainOnePlan(a,b,c,d,e,f,g)
+#define Set_ptr_value(a,b) a->data.ptr_value = b
```

Comment:
5a7c2c1

The others seem to be not used.

Reply:
I think we can leave it out for now. Might have been easier to add the macro's now, even if we don't need them today.

Once someone needs them they need to add the macros here in a consistent way to keep the code maintainable. Downside would be to add dead code, but I think for these lines it would have been ok.

Due to the complexities of rebasing and enterprise we can skip it for now.

If you were to revisit here, would it make sense to apply `5a7c2c1` as a fixup?

Reply:
Putting the same comment here:

I have tried rebasing with that commit actually but it creates many conflicts while going over the other commits mostly because the style fix commit is a separate one